stable/hazelcast/.helmignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
Please use the `.helmignore` file you get from `helm create`.

Reply:
Done